hack/build/docker/builder/Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN \
     mkdir -p ${GOPATH} && \
     source /etc/profile.d/gimme.sh && \
     eval $(go env) && \
+    go get -u github.com/onsi/ginkgo/ginkgo && \
```

Comment:
does this need to be `go install ./...`'d ?

Reply:
Good point, all the others are, fixed.